src/main/java/org/openpnp/machine/reference/vision/wizards/ReferenceFiducialLocatorConfigurationWizard.java

```diff
@@ -0,0 +1,233 @@
+package org.openpnp.machine.reference.vision.wizards;
```

Comment:
Right now template detection is working so well, I don't know if we need the keypoint code.  The code is intended to allow maximum flexibility in setting up a pipeline, but provide sensible working defaults if not needed, so yeah, user shouldn't worry about this (but some of us do).

Yes, single set of results can be named 'results', but then can't show an example of both matching types inside the configuration, which is fine.  If there were a CvStage class which amalgamated the different detection types, it would be different.  I can do this, second bullet is right idea.